static/css/components/jquery.autocomplete.less

```diff
@@ -3,7 +3,7 @@
 /* stylelint-disable no-descending-specificity */
 .ac_results {
   padding: 0;
-  overflow: hidden;
+  overflow: scroll;
```

Comment:
This should avoid cases like this:
![image](https://user-images.githubusercontent.com/6251786/109386312-d908f500-78c7-11eb-8f4f-0e6b3b419779.png)


```suggestion
  overflow: auto;
```

Reply:
fixed